cmd/generate/generate.go

```diff
@@ -225,16 +226,37 @@ func main() {
 				projectWithSep = capitalize(projectName) + " "
 				projectDashCaseWithSep = projectName + "-"
 			}
+
+			const additionalInstructions = "additional_instructions"
+
 			d := map[string]interface{}{
-				"main":                p,
-				"app_file":            appFile,
-				"builder":             builderImage,
-				"version":             metadata.Project.Tag,
-				"project":             projectWithSep,
-				"project_dashcase":    projectDashCaseWithSep,
-				"component":           capitalize(p),
-				"component_dashcase":  dashcase(p),
-				"additional_packages": strings.Join(additionalPackages, " "),
+				"main":                 p,
+				"app_file":             appFile,
+				"builder":              builderImage,
+				"version":              metadata.Project.Tag,
+				"project":              projectWithSep,
+				"project_dashcase":     projectDashCaseWithSep,
+				"component":            capitalize(p),
+				"component_dashcase":   dashcase(p),
+				additionalInstructions: []string{},
+				"additional_packages":  strings.Join(additionalPackages, " "),
+			}
+			if strings.Contains(p, "ping") || slices.Contains(additionalPackages, "tzdata") {
```

Comment:
What exactly is the meaning of the `strings.Contains(p, "ping")` ?

Reply:
pingsource it's a cmd directory called `mtping`

Reply:
ok, so this would only be added into the mtping image?  (I think the eventing-webhook might need it as well.)

Reply:
Good point, I'll add it through flag and remove this special casing here